Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38462&attempt_id=d2e740c5-2653-44c1-8f4d-fc58860ddb0d)

Attempted: tome4

<details><summary>Partial log (click to expand)</summary><p>

```
configuring
../src/luajit2/src/lj_arch.h:55:2: error: #error "No support for this architecture (yet)"
 #error "No support for this architecture (yet)"
  ^~~~~
../src/luajit2/src/lj_arch.h:272:2: error: #error "No target architecture defined"
 #error "No target architecture defined"
  ^~~~~
/build/t-engine4-src-1.5.5/build/te4core.lua:214: Unsupported target architecture, use architecture agnostic lua with --lua=default
builder for '/nix/store/2riz3qc1913gcyaspkz872l1ysgcagb4-tome4-1.5.5.drv' failed with exit code 1
[31;1merror:[0m build of '/nix/store/2riz3qc1913gcyaspkz872l1ysgcagb4-tome4-1.5.5.drv' failed
```
</p></details>

--- a/pkgs/games/tome4/default.nix
+++ b/pkgs/games/tome4/default.nix
@@ -82,6 +82,6 @@ in stdenv.mkDerivation rec {
     homepage = https://te4.org/;
     license = licenses.gpl3;
     maintainers = with maintainers; [ chattered peterhoeg ];
-    platforms = platforms.linux;
+    platforms = subtractLists ["aarch64-linux"] platforms.linux;
   };
 }

